cmd/describe/cluster/cmd.go

```diff
@@ -335,6 +286,74 @@ func run(cmd *cobra.Command, argv []string) {
 	fmt.Println()
 }
 
+func controlPlaneConfig(cluster *cmv1.Cluster) string {
+	if cluster.Hypershift().Enabled() {
+		return "Red Hat hosted"
+	}
+	return "Customer hosted"
+}
+
+func clusterInfraConfig(cluster *cmv1.Cluster, clusterKey string, r *rosa.Runtime) string {
+	var infraConfig string
+	if cluster.Hypershift().Enabled() {
+		infraConfig = fmt.Sprintf(""+
+			"Node Pools:\n"+
+			" - Compute:                 %d\n",
+			cluster.Nodes().Compute(),
+		)
+	} else {
+		// Display number of all worker nodes across the cluster
+		minNodes := 0
+		maxNodes := 0
+		machinePools, err := r.OCMClient.GetMachinePools(cluster.ID())
+		if err != nil {
+			r.Reporter.Errorf("Failed to get machine pools for cluster '%s': %v", clusterKey, err)
+			os.Exit(1)
+		}
+		// Accumulate all replicas across machine pools
+		for _, machinePool := range machinePools {
+			if machinePool.Autoscaling() != nil {
+				minNodes += machinePool.Autoscaling().MinReplicas()
+				maxNodes += machinePool.Autoscaling().MaxReplicas()
+			} else {
+				minNodes += machinePool.Replicas()
+				maxNodes += machinePool.Replicas()
+			}
+		}
+		// Add compute nodes as well
+		if cluster.Nodes().AutoscaleCompute() != nil {
+			minNodes += cluster.Nodes().AutoscaleCompute().MinReplicas()
+			maxNodes += cluster.Nodes().AutoscaleCompute().MaxReplicas()
+		} else {
+			minNodes += cluster.Nodes().Compute()
+			maxNodes += cluster.Nodes().Compute()
+		}
+		// Determine whether there is any auto-scaling in the cluster
+		if minNodes == maxNodes {
+			infraConfig = fmt.Sprintf(""+
+				"Nodes:\n"+
+				" - Control plane:           %d\n"+
+				" - Infra:                   %d\n"+
```

Comment:
I think these components can be hidden, but everything else should still be in place (but instead of `GetMachinePools` you should call `GetNodePools` or the equivalent function). No need for a large `if hypershift {...} else {...}`

Reply:
The key is also different; there is no `Nodes` but `Node Pools` instead. We need to refactor this once the nodepool is in db, and nodepool controller is in place.